lib/sycamore/sycamore/tests/unit/scans/test_file_scan.py

```diff
@@ -58,6 +67,17 @@ def test_json_scan_all_props(self):
         assert doc.properties["props"] == "propValue"
         assert isinstance(doc.properties["web-app"], dict)
 
+    def test_json_scan_all_props_local(self):
```

Comment:
Do we not have tests for JsonDocumentScan?

Reply:
no, we don't have. Test against the notebook having jsondocumentscan